data/ch/banner.yml

```diff
@@ -0,0 +1,12 @@
+############################### Banner ################################
+banner:
+  enable : true
+  bg_image_webp : "images/backgrounds/NASA_GulfofMex.webp"
+  bg_image : "images/backgrounds/NASA_GulfofMex.jpg"
+  icon : "" # themify icon pack : https://themify.me/themify-icons
+  title : "Rotational Labs"
+  content : "Réinventer le possible pour vos apps"
+  button:
+    enable : true
+    label : "Explorer"
+    link : "#blog"
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)